### ISSUE TITLE
CVE-2023-24538 Vulnerability 

### ISSUE THREAD

**Issue by VDN1** [GoogleContainerTools/kaniko#2560]
**Actual behavior**
Please address CVE-2023-24538, CVE-2022-27664 and CVE-2022-41723 vulnerabilities in  gcr.io/kaniko-project/executor:latest image

**Expected behavior**

**To Reproduce**
1. ...
2. ...

**Additional Information**
 - Dockerfile 
   Please provide either the Dockerfile you're trying to build or one that can reproduce this error.
 - Build Context
   Please provide or clearly describe any files needed to build the Dockerfile (ADD/COPY commands)
 - Kaniko Image (fully qualified with digest)
 
 **Triage Notes for the Maintainers**
 

 
 | **Description** | **Yes/No** |
 |----------------|---------------|
 | Please check if this a new feature you are proposing        | <ul><li>- [ ] </li></ul>|
 | Please check if the build works in docker but not in kaniko | <ul><li>- [ ] </li></ul>| 
 | Please check if this error is seen when you use `--cache` flag | <ul><li>- [ ] </li></ul>|
 | Please check if your dockerfile is a multistage dockerfile | <ul><li>- [ ] </li></ul>|

**Comment by VDN1** [comment]
Is there any ETA with new images without mentioned vulnerabilities.

**Comment by aaron-prindle** [comment]
The current latest release of Kaniko has the following vulns as scanned by `grype` (not sure how they map to those identified above):
```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:latest
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.62.3 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [203 packages]
 ✔ Scanning image...       [11 vulnerabilities]
   ├── 0 critical, 6 high, 5 medium, 0 low, 0 negligible
   └── 9 fixed
NAME                      INSTALLED                           FIXED-IN                           TYPE       VULNERABILITY        SEVERITY 
github.com/docker/docker  v20.10.12+incompatible              20.10.24                           go-module  GHSA-232p-vwff-86mp  High      
github.com/docker/docker  v20.10.12+incompatible              20.10.24                           go-module  GHSA-33pg-m6jh-5237  Medium    
github.com/docker/docker  v20.10.12+incompatible              20.10.24                           go-module  GHSA-6wrf-mxfj-pf5p  Medium    
golang.org/x/net          v0.0.0-20220127200216-cd36cc0744dd  0.0.0-20220906165146-f3363e06e74c  go-module  GHSA-69cg-p879-7622  High      
golang.org/x/net          v0.0.0-20220127200216-cd36cc0744dd  0.7.0                              go-module  GHSA-vvpx-j8f3-3w6h  High      
golang.org/x/sys          v0.0.0-20220114195835-da31bd327af9  0.0.0-20220412211240-33da011f77ad  go-module  GHSA-p782-xgp4-8hr8  Medium    
golang.org/x/sys          v0.0.0-20220325203850-36772127a21f  0.0.0-20220412211240-33da011f77ad  go-module  GHSA-p782-xgp4-8hr8  Medium    
golang.org/x/text         v0.3.6                              0.3.7                              go-module  GHSA-ppp9-7jff-5vj2  High      
golang.org/x/text         v0.3.6                              0.3.8                              go-module  GHSA-69ch-w2m2-3vjp  High      
```

At HEAD though, these have all been patched/fixed (as of 6/20/2023):
```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:latest
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.62.3 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [202 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

The next release of Kaniko, `v1.12.0`, is scheduled to go out 6/27/2023 - https://github.com/GoogleContainerTools/kaniko/milestone/4

**Comment by aaron-prindle** [comment]
This is resolved now with the release today of `v1.12.0`:

```
aprindle@aprindle-ssd ~/kaniko  [main]grype gcr.io/kaniko-project/executor:v1.12.0
 ✔ Vulnerability DB        [no update available]
New version of grype is available: 0.63.0 (currently running: 0.61.0)
 ✔ Loaded image            
 ✔ Parsed image            
 ✔ Cataloged packages      [202 packages]
 ✔ Scanning image...       [2 vulnerabilities]
   ├── 0 critical, 1 high, 1 medium, 0 low, 0 negligible
   └── 0 fixed
No vulnerabilities found
```

Closing
